boards/DCM/Inc/App/configs/App_MotorEfficiencyLut.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#define MOTOR_EFFICIENCY_LUT_INTERVAL (500U)
+#define MOTOR_EFFICIENCY_LUT_SIZE (14U)
+
+// Motor efficiency LUT with interval of 500rpm
+static const float motor_efficiency_lut[MOTOR_EFFICIENCY_LUT_SIZE] = { 0.8f, 0.8f, 0.8f, 0.8f, 0.8f, 0.8f, 0.8f,
```

Comment:
@timbitss if we calculate the index, we wont need the speed LUT. Thus an array of float will do

Reply:
also figured we should rename the file to be more specific. Didn't rlly know what the efficiency was for

Reply:
I agree. I had left the speed in there, however, since it was nice to quickly know what speed mapped to what efficiency.

Reply:
Moreso efficiency of the inverter + motor but idc